ivadomed/inference.py

```diff
@@ -225,7 +225,9 @@ def pred_to_png(pred_list: list, target_list: list, subj_path: str, suffix: str
     for pred, target in zip(pred_list, target_list):
         filename = subj_path + target + suffix
         data = pred.get_fdata()
-        imageio.imwrite(filename, (data*255/max_value).astype(np.uint8), format='png')
+        _img = (data*255/max_value).astype(np.uint8)
+        img = np.dstack((_img, _img, _img))
```

Comment:
You don't need to do a depth stack here unless you want the image to be stored as a RGB image. 

If you pass a grayscale image (shape `(height, width)`) directly then ImageIO will store it as a grayscale image, which (a) is smaller, and (b) will be read as a grayscale image :)

Reply:
@FirefoxMetzger thank you for this catch. You are right and the depth stack was also causing a bug for some images.